src/components/kpi/kpi.less

```diff
@@ -154,13 +154,30 @@
       text-align: center;
     }
 
-    .im-button.alert-caption {
+    .alert-caption {
       text-align: center;
       font-size: 11px;
-      width: 81%;
+      background: lighten(@kpi-negative-alert-label-background-color, 20%);
+      border-radius: 3px;
+      width: 100%;
       margin: 0 auto 5px auto;
       box-shadow: none;
       padding: 2px;
+      color: white;
+    }
+
+    .im-button.alert-config-btn {
```

Comment:
not too sure about this... have you checked it's still working on mobile?
I let @xaun judge for the color and layout

Generally speaking I would keep this kind of change in a separate pull request as it can be subject to discussion and block the other fixes your PR is aiming to resolve...